### ISSUE TITLE
Information in /Doc outdated

### ISSUE THREAD

**Issue by Stewori** [jython/jython#180]
This is low priority, I only open it as a reminder. I recently noticed the [/Doc folder](https://github.com/jython/jython/tree/master/Doc) which contains awfully outdated information. The files look to me as if they once were part of a generated documentation. OpenJDK source also has a /doc folder; maybe there was a config to include them in javadoc or so. Not all info there is bad or outdated and I am not suggesting to delete the folder. If I find time I can update it and remove the hopeless files (e.g. about jythonc). Does someone know what the ``.ht`` ending is about? It looks like ``<head>``-less html. Is this some standard or understood by some tool?

**Comment by jeff5** [comment]
It may have generated a website, by a lost process traces of which are found in [/Misc/htgen](https://github.com/jython/jython/tree/master/Misc/htgen).

**Comment by fwierzbicki** [comment]
The /Doc folder is indeed completely outdated. It did (once) generate a website. I think when I first started contributing it still worked, but I moved to some kind of restructured text version in the Jython 2.2 timeframe and ported over what info I could use. I think all of that documentation is so outdated now that it's probably just a distraction to new users looking around the repo, I'd vote for removing it since we have plenty of much better documentation now.
